### PR TITLE
Bugfix/cldsrv 286 bump arsenal fix authz regression

### PR DESCRIPTION
**Pull request by ghost**
Bump Arsenal in Cloudserver. It fixes the issue described in https://github.com/scality/Arsenal/pull/2016.

I also bump the dockerfile base image version, to get rid of some CVEs.

Zenko build with the changes: https://github.com/scality/Zenko/actions/runs/3173170506

### CODE REVIEW

**Review by welansari — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zenko/cloudserver",
-  "version": "8.4.16",
+  "version": "8.4.15",
```
Comment (jonathan-gramain) [inline package.json]: Is it wanted to decrement the version?

Comment (ghost) [reply package.json]: No, indeed, thank you! I fixed it, and I will fix the conflicts in 8.5 branch right away. In 8.5 the version will be bumped as I will release the project after, for Artesca.

Comment (ghost) [reply package.json]: Done. Here is the link: https://github.com/scality/cloudserver/compare/w/8.5/bugfix/CLDSRV-286-bump-arsenal-fix-authz-regression

**Review by jonathan-gramain — APPROVED** [review]

**Pull request merged**